lib/android/src/main/java/com/reactnativeldk/classes/CustomKeysManager.kt

```diff
@@ -0,0 +1,89 @@
+package com.reactnativeldk.classes
+
+import org.ldk.structs.KeysManager
+import org.ldk.structs.Option_u32Z
+import org.ldk.structs.Result_CVec_u8ZNoneZ
+import org.ldk.structs.Result_ShutdownScriptInvalidShutdownScriptZ
+import org.ldk.structs.Result_ShutdownScriptNoneZ
+import org.ldk.structs.Result_TransactionNoneZ
+import org.ldk.structs.Result_WriteableEcdsaChannelSignerDecodeErrorZ
+import org.ldk.structs.ShutdownScript
+import org.ldk.structs.SignerProvider
+import org.ldk.structs.SignerProvider.SignerProviderInterface
+import org.ldk.structs.SpendableOutputDescriptor
+import org.ldk.structs.TxOut
+import org.ldk.structs.WriteableEcdsaChannelSigner
+import org.ldk.util.UInt128
+import org.ldk.util.WitnessVersion
+
+class CustomKeysManager(
+    seed: ByteArray,
+    startingTimeSecs: Long,
+    startingTimeNanos: Int,
+    val destinationScriptPublicKey: ByteArray,
+    val witnessProgram: ByteArray,
+    val witnessProgramVersion: Byte
+) {
+    val inner: KeysManager = KeysManager.of(seed, startingTimeSecs, startingTimeNanos)
```

Comment:
Another random thought I like to share, definitely no change needed:
I like to wrap property inits in a `by lazy { __initBlock }`, it provides a bit more "safety" imho since it would only really create the value at the first call time.

E.g.
```kt
val inner: KeysManager by lazy { KeysManager.of(seed, startingTimeSecs, startingTimeNanos) }
```